packages/dm-core-plugins/src/attribute-selector/AttributeSelectorPlugin.tsx

```diff
@@ -90,6 +90,7 @@ export const AttributeSelectorPlugin = (props: IUIPlugin): JSX.Element => {
         style={{
           display: 'flex',
           flexDirection: config.asSidebar ? 'row' : 'column',
+          width: '-webkit-fill-available',
```

Comment:
```suggestion
```

Reply:
This one is needed. If not, the "fill avilable" only effects the first level of the sidebar (if they are nested).
But "100%" works on chrome and firefox :+1: